internetarchive/bookreader
# interstitial popup: clean up css so that it flows with parent container

This PR cleans up css and hard positioning on progress modals.
Now, it flows with the parent container.

To test:
- pull down branch, run `npm install && npm run serve-dev` to run demo server & compile latest code.
- go to Internet Archive demo -> search 'virtue'
- watch progress bar come into view
- shrink & grow browser window and confirm that progress popup does not leave the center area of the bookreader

Or go to www-isa.archive.org/details/goody?q=catches
- when running search on page load, confirm that the progress popup stays inside of the bookreader container & does not bleed out of screen


![ezgif com-gif-maker](https://user-images.githubusercontent.com/7840857/94861895-16d32180-03ed-11eb-93b2-b46054587412.gif)
